src/Pages/LandingPage.js

```diff
@@ -1,19 +1,18 @@
 import React from 'react'
 import './LandingPage.css'
-import FBIcon from '../Assets/Images/facebook-icon.svg'
-import LinkedInIcon from '../Assets/Images/linkedin-icon.svg'
-import TwitterIcon from '../Assets/Images/twitter-icon.svg'
-import GitHubIcon from '../Assets/Images/github-icon.svg'
+import LpBackground from '../Components/LpBackground'
+import SocialLinks from '../Components/SocialLinks'
+import LpHeader from '../Components/LpHeader'
+import LpButton from '../Components/LpButtonSection'
+
 
 function LandingPage() {
   return (
-    <div className='landing-page-container'>
-      <div className="social-icons-container">
-        <img src={FBIcon} alt="" className="si1" />
-        <img src={LinkedInIcon} alt="" className="si2" />
-        <img src={TwitterIcon} alt="" className="si3" />
-        <img src={GitHubIcon} alt="" className="si4" />
-      </div>
+    <div>
+      <LpBackground />
+      <SocialLinks />
+      <LpHeader />
+      <LpButton />
     </div>
```

Comment:
Use a fragment tag here:

https://reactjs.org/docs/fragments.html

```suggestion
    <>
      <LpBackground />
      <SocialLinks />
      <LpHeader />
      <LpButton />
    </>
```

Reply:
Oh cool! I have seen that done. I will go through and fix all of that.

Reply:
I was the one that was trying to tell you to do that lol

Reply:
Yes you did T!